index1.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+
+
+<body bgcolor=white>
+    
+
+<section>
+    <img src="feedback-banner.jpg" style="width:100%;height:500px;">
+    <form action="/action_page.php">
+ Username:<br>
+  <input type="text" name="username" value="">
+  <br>
+ Password:<br>
+  <input type="text" name="password" value="">
+  <br><br>
+  <input type="submit" value="Login">
+</form> 
+    
+    
+  </section>
+    <section  style="display:inline-flex">
+        <img src="new feedback.jpg" width="15%" height="300px" >
+        
+      
+   <article >
+         <h3> Submit new feedback
```

Comment:
As a best practice, a document should include all heading levels, in their proper order. `<h3>` should be preceded by `<h2>`, which should be preceded by `<h1>`.